src/printer.js

```diff
@@ -2344,13 +2344,28 @@ function printNode(path, options, print) {
       );
     case "variable": {
       const parent = path.getParentNode();
+      const parentParent = path.getParentNode(1);
```

Comment:
Nitpick: Would “grandparent” be a better name?

Reply:
We already used `parentParent` and `parentParentParent` and etc in our code so i think better keep this name as is

Reply:
Alright!